### PR TITLE
Clean coredump info for cases generating coredump during test

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

File: libvirt/tests/src/daemon/kill_qemu.py [hunk 1]
```diff
@@ -14,6 +15,7 @@ def run(test, params, env):
     sig_name = params.get('signal', 'SIGSTOP')
     vm_state = params.get('vm_state', 'running')
     expect_stop = params.get('expect_stop', 'yes') == 'yes'
+    expect_coredump = params.get('expect_coredump', 'yes') == 'yes'
```
Comment (Yingshun) [inline libvirt/tests/src/daemon/kill_qemu.py:18]: The default value of 'expect_coredump' should be 'no', right?

Comment (yafu-1) [reply libvirt/tests/src/daemon/kill_qemu.py]: Modified.

Comment (yafu-1) [reply libvirt/tests/src/daemon/kill_qemu.py]: Modified.

**Review by Yingshun — APPROVED** [review]

**Review by brucepatricklee — APPROVED** [review]

**Review by chloerh — APPROVED** [review]
Run all cases with this pr, all passed. Thank you for the great fix!

**Pull request merged**